src/h5web/visualizations/line/DataLine.tsx

```diff
@@ -1,15 +1,22 @@
-import React, { useCallback } from 'react';
+import React from 'react';
+import { useUpdate } from 'react-three-fiber';
+import { BufferGeometry } from 'three';
 import { useDataPoints } from './hooks';
 
 function DataLine(): JSX.Element {
   const points = useDataPoints();
-  const setFromPoints = useCallback(self => self.setFromPoints(points), [
-    points,
-  ]);
+  const ref = useUpdate(
+    (geometry: BufferGeometry) => {
+      if (points) {
+        geometry.setFromPoints(points);
+      }
+    },
+    [points]
+  );
```

Comment:
This was changed as the line plot was not updating whereas `points` was changed.